### PR TITLE
Use `singularity` containers for the injection script

### PR DESCRIPTION
**Pull request by ggonzr**
Fixes: #1119

### CODE REVIEW

File: mcm/tools/handlers.py [hunk 1]
```diff
@@ -310,7 +310,10 @@ def make_injection_command(self, mcm_r=None):
         command += 'export X509_USER_PROXY=%s\n\n' % (proxy_file_name)
 
         executable_file_name = '%supload_script_%s.sh' % (directory, mcm_r.get_attribute('prepid'))
-        default_scram_arch = scram_arch.startswith('slc7_')
+
+        # Always use singularity containers.
+        default_scram_arch = False
```
Comment (lmoureaux) [inline mcm/tools/handlers.py:315]: Reduce complexity by removing the variable altogether

**Review by lmoureaux — CHANGES_REQUESTED** [review]

**Comment by lmoureaux** [comment]
Guess this can be closed as the same patch was implemented in #1120 ?

**Comment by lmoureaux** [comment]
> Guess this can be closed as the same patch was implemented in https://github.com/cms-PdmV/cmsPdmV/pull/1113 ?

Sorry for the noise, 1113 was about a different file.

File: mcm/tools/handlers.py (hunk 1 above)
Comment (ggonzr) [reply mcm/tools/handlers.py]: Fixed in the latest commit

**Review by lmoureaux — APPROVED** [review]

**Pull request merged**